File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/DonorResponseDto.java
package com.sahayaSetu.dtos;

import lombok.Data;
import java.time.LocalDate;

@Data
public class DonorResponseDto {
    private Long donorId;
    private String fullName;
    private String email;
    private String phone;
    private String city;
    private LocalDate dateOfBirth;
}

File: backend-dotnet/SahayaSetu/DTO/DonorResponseDto.cs
namespace SahayaSetu.DTO
{
    public class DonorResponseDto
    {
        public int DonorId { get; set; }
        public string? FullName { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? City { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
